ts/components/messages/paginated/MessageList/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { none, Option, some } from "fp-ts/lib/Option";
 import * as pot from "italia-ts-commons/lib/pot";
+import { Millisecond } from "italia-ts-commons/lib/units";
```

Comment:
```suggestion
import { Millisecond } from "@pagopa/ts-commons/lib/units";
```

since we aim to migrate it let's use the pagopa lib version if you don't mind